client/src/components/Form/components/formSections/MulighetForArbeidSection.tsx

```diff
@@ -49,17 +51,20 @@ type MulighetForArbeidSectionProps = {
 
 const MulighetForArbeidSection = ({ section, setSchema, schema, errors, validate }: MulighetForArbeidSectionProps) => {
     return (
-        <SectionContainer section={section}>
+        <SectionContainer section={section} sectionError={errors.mulighetForArbeid}>
             <Subsection sectionIdentifier="4.1">
                 <Checkbox
                     checked={schema.avventendeSykmelding}
                     label="Pasienten kan benytte avventende sykmelding"
                     onChange={() => {
-                        validate('avventendeSykmelding', !schema.avventendeSykmelding);
-                        setSchema(state => ({
-                            ...state,
-                            avventendeSykmelding: !state.avventendeSykmelding,
-                        }));
+                        setSchema(state => {
+                            validate('avventendeSykmelding', !schema.avventendeSykmelding);
```

Comment:
```suggestion
                            validate('avventendeSykmelding', !state.avventendeSykmelding);
```

Her bør vi validere på det som faktisk skal lagres i skjemaet. Verdien vil nok være det samme i både schema og state, men det er en mikroskopisk risiko her om vi validerer på schema.avventendeSykmelding i stedet for state.avventendeSykmelding.

Dette gjelder forøvrig alle validate()-funksjoner som ligger inne i en setSchema(state =>